cli/command/new/cmd.go

```diff
@@ -34,6 +35,14 @@ var (
 				Default: "Cypress",
 			},
 		},
+		{
+			Name: "platform",
+			Prompt: &survey.Select{
+				Message: "Choose a platform:",
+				Options: []string{"Win 10", "Mac 11.00"},
```

Comment:
Are those values translated somewhere to `Windows 10` and `MacOS 11.0` ?
Otherwise, I think template will fail when allocating a VM because of un-existing platform.

Reply:
agree